src/arch/helperavx.h

```diff
@@ -29,6 +29,16 @@
 
 #define FULL_FP_ROUNDING
 
+#ifdef SLEEF_SINGLE_MINMAXNUM_AVAILABLE
```

Comment:
I must have been very unclear in my requests in #106 ! :)

Given that you have added the generic implementation of `v[max|min]num_vd_vd_vd` in ``dd.h``, guarded with an `#ifndef`, there is no need to provide it via the helper header files. You should just be able to revert the `avx*` header file changes you have introduced in this patch, so that the generic implementation in `dd.h` would be picked up. A similar reasoning holds for the single precision version of the functions.

So please remove the macros , and the generic implementations from the `axv*` header files.